metis-enrichment/metis-enrichment-proxy-rest/src/main/java/eu/europeana/enrichment/cache/proxy/config/Application.java

```diff
@@ -68,7 +68,10 @@ EnrichmentEntityDao getEntityDao() {
 
   @Bean(name = "redisInternalEnricher")
   RedisInternalEnricher getRedisInternalEnricher() {
-    return new RedisInternalEnricher(getEntityDao(), getRedisProvider(), true);
+    final RedisInternalEnricher enricher = new RedisInternalEnricher(getEntityDao(),
+            getRedisProvider());
+    new Thread(enricher::recreateCache).start();
```

Comment:
I think this shouldn't be executed on every redeployment/restart.
We should have something like a rest call that enables a flag and enforces the recreation on the next restart.

Reply:
OK, I fixed it.